### DISCUSSION TITLE
Repositories metrics has changed

### DISCUSSION THREAD

**Discussion by andry81** [DenverCoder1/github-readme-streak-stats#280] [Q&A] [answered]
https://github.com/andry81-cache/andry81--gh-content-cache/commit/4eef63ed69ca4299358812e7ffe650383562e1fc#diff-566213d7bf64064d44a7e8b161368c146b7d1d07e403356652f97f7aa40abbd3

Changed from 2.21 GB to 75.9 MB.

As I understand, is the new size without forks?

**Comment by DenverCoder1** [comment, marked as answer]
Github-readme-streak-stats shows only contribution counts, not repository sizes, maybe you meant to open this in https://github.com/lowlighter/metrics?

**Reply by andry81** [reply]
ops, misclick
